DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DelegateGroupsController.cs

```diff
@@ -105,14 +107,15 @@ public IActionResult GroupDelegatesRemove(int groupId, int delegateId)
         {
             var centreId = User.GetCentreId();
             var groupName = groupsDataService.GetGroupName(groupId, centreId);
-            var delegateUser = userDataService.GetDelegateUserById(delegateId);
-            var groupDelegates = groupsDataService.GetGroupDelegates(groupId);
+            var groupDelegates = groupsDataService.GetGroupDelegates(groupId).ToList();
 
-            if (delegateUser == null || groupName == null || groupDelegates.All(gd => gd.DelegateId != delegateId))
+            if (groupName == null || groupDelegates.All(gd => gd.DelegateId != delegateId))
             {
                 return NotFound();
             }
 
+            var delegateUser = groupDelegates.Single(gd => gd.DelegateId == delegateId);

```

Comment:
Minor, but we could do this before "NotFound" and use "delegateUser == null" check instead of doing .All